tests/unittests/tiglWingCSTProfile.cpp

```diff
@@ -242,6 +289,23 @@ TEST(CSTApprox, hypersonicAirfoil)
             devmax=deviation;
         }
     }
-    // approximation should be exact since we require only degree 2 spline
-    ASSERT_NEAR(0.0, devmax, 1e-12);
+
+    if (algo == tigl::CCSTCurveBuilder::Algorithm::Piecewise_Chebychev_Approximation) {
+        // approximation should be exact since we require only degree 2 spline
+        ASSERT_NEAR(0.0, devmax, 1e-12);
+    }
+
+    if (algo == tigl::CCSTCurveBuilder::Algorithm::GeomAPI_PointsToBSpline) {
+        // I doubt the optimzation algo used by GeomAPI_PointsToBSpline guarantees the exact solution.
+        ASSERT_NEAR(0.0, devmax, 1e-5);
+    }
 }

```

Comment:
I like this parametric testing setup, although here, we only have 2 parameters - still it makes the test thorough and the code shorter.

I think these tests are enough and one does not need to test the curvature of the curves explicitly (which was the main concern of this implementation - to avoid curvature jumps), as the curves created with the algorithm GeomAPi_PointsToBSpline anyway create B-spline curves of degree 4, which guarantee C3-continuity along the whole curve.